### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by rbunn80110** [muellerzr/fastinference_pytorch#14]
There is a notice of this project being on hold pending updates.  Any ETA on that?

**Comment by muellerzr** [comment]
It's on hold for *now*. Jeremy is wanting to improve inference on the fastai side, so once that is done I plan on updating this. My main issue I saw while doing this was porting over transforms. With how the fastai api is now, it's no longer just an "add-on" for pytorch, so everything is integrated, even the transforms. 

I had to really think about if I wanted to recreate every fastai transform to make it work in raw pytorch, but that seemed like a mistake to me. Thank you for bringing this back up though, as it is definitely important! I'm not abandoning this in the slightest though, so do not worry about that.

**Comment by rbunn80110** [comment]
Awesome!  Keep up the great work!

**Comment by muellerzr** [comment]
Thank you! (Also leaving this open as a reminder to me to actually work on this over the semester break :) )
